### DISCUSSION TITLE
Error: Cannot find module '‘global-agent/bootstrap’'

### DISCUSSION THREAD

**Discussion by Adem-Kri** [eclipse-theia/theia#11800] [General] [answered]
I am trying to build theia project using yarn but am facing 2 issues.

I cloned Theia repo but when building using yarn 

```
kriouana@TUN MINGW64 ~/Desktop/theia (master)
$ yarn
```
I got this error related to some file called INSTALL.JS

```
error C:\Users\kriouana\Desktop\theia\node_modules\electron: Command failed.
Exit code: 1
Command: node install.js
Arguments:
Directory: C:\Users\kriouana\Desktop\theia\node_modules\electron
Output:
RequestError: getaddrinfo ENOTFOUND github.com
```
So after searching about this issue I found this

```
yarn global add global-agent
cd <clone_dir>/theia/node_modules/electron
node -r ‘global-agent/bootstrap’ install.js
```
but when executing the last line this came up

```
$ node -r ‘global-agent/bootstrap’ install.js
node:internal/modules/cjs/loader:988
  throw err;
  ^

Error: Cannot find module '‘global-agent/bootstrap’'
Require stack:
- internal/preload
```

its something related to node, install.js or global-agent

**Comment by vince-fugnitto** [comment]
@Adem-Kri thank you for the discussion, can you confirm that you have satisfied all [prerequisites](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites) in order to build?

I've never encountered build issues where `global-angent/bootstrap` had be explicitly added, do you happen to be behind a proxy (it doesn't seem to be able to connect to `github.com`)?

**Reply by Adem-Kri** [reply]
@vince-fugnitto yes I am behind my company's proxy but I have set the hhtp_proxy, https_proxy and global_agent_http_proxy.

and I am sure that all prerequisites are satisfied

**Reply by vince-fugnitto** [reply]
I believe the error ultimately comes down to `@electron/get` (which uses `global-agent` itself), perhaps the documentation it provides for [proxy support](https://github.com/gajus/global-agent/blob/v2.1.5/README.md#environment-variables) would be useful. The documentation is also linked from [electron](https://www.electronjs.org/docs/latest/tutorial/installation#proxies) itself.

**Comment by Adem-Kri** [comment, marked as answer]
I have already found the solution just I wrote the wrong quote `‘ ’` when I should use `' '`,
```
node -r 'global-agent/bootstrap' install.js
```
